runtime/src/main/jni/NetworkDomainCallbackHandlers.h

```diff
@@ -0,0 +1,278 @@
+//
+// Created by pkanev on 3/8/2017.
+//
+
+#ifndef NETWORKDOMAINCALLBACKHANDLERS_H
+#define NETWORKDOMAINCALLBACKHANDLERS_H
+
+
+#include <include/v8.h>
+#include <v8_inspector/src/inspector/v8-network-agent-impl.h>
+#include "JsV8InspectorClient.h"
+#include "ArgConverter.h"
+
+namespace tns {
+namespace NetworkDomainCallbackHandlers {
+
+static const char* FrameId = "NSFrameIdentifier";
+static const char* LoaderId = "NSLoaderIdentifier";
+
+static void ResponseReceivedCallback(const v8::FunctionCallbackInfo<v8::Value>& args) {
```

Comment:
Callbacks have since been documented. I'll just extract them as static members of a new class now.